src/data/wgs/embedded/team.yml

```
@@ -4,3 +4,31 @@ description: Focusing on the end-to-end experience of using Rust in resource-con
 chat: Domain Working Groups #wg-embedded channel on Discord
 repo: "https://github.com/rust-embedded/"
 leads: [japaric]
+members: [
+  adamgreig,
+  andre-richter,
+  awygle,
+  bradjc,
+  cr1901,
+  danc86,
+  dvc94ch,
+  dylanmckay,
+  Emilgardis,
+  hannobraun,
+  ithinuel,
+  jamesmunns,
+  jcsoo,
+  korken89,
+  nastevens,
+  nerdyvaishali,
+  paoloteti,
+  parched,
+  pftbest,
+  posborne,
+  raw-bin,
+  ryankurte,
+  sekineh,
+  thejpster,
+  therealprof,
+  wizofe,
```

There is a trailing comma here, the other examples of members list omit the extra comma. Perhaps that why the build failed
```suggestion
  wizofe
```